promptops/promptops/skill_export.py

```
@@ -144,22 +144,25 @@ def render_skill(raw_content: str, raw_data: Any) -> tuple:
         })
     return (skill_content, errors)
 
-def write_skill_file(prompts_path: Path, docs_path: Path, path: Path, skill_content: str):
+def write_skill_file(prompts_path: Path, docs_path: Path, path: Path, skill_content: str, reconciler: 'DirectoryReconciler' = None):
```

Ruff (0.15.15)

[error] Undefined name `DirectoryReconciler`

(F821)